incubator/domoticz/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+name: domoticz
+description: A Helm chart for Kubernetes
+version: 0.1.0
+home: https://www.domoticz.com/
```

Comment:
Can you please add the appropriate metadata for maintainer, source, icon and optionally some keywords?